### ISSUE TITLE
Loss should be real - fake in Tutorial 8

### ISSUE THREAD

**Issue by najwalb** [phlippe/uvadlc_notebooks#60]
**Tutorial**: 8

**Describe the bug**
The loss function used in the implementation of 'DeepEnergyModel()' in tutorial 8 (cdiv_loss) does not match the loss described earlier in the algorithm. It should be: cdiv_loss = real_out.mean() - fake_out.mean(), not the other way around.

**To Reproduce (if any steps necessary)**
1. Go to [Tutorial 8](https://uvadlc-notebooks.readthedocs.io/en/latest/tutorial_notebooks/tutorial8/Deep_Energy_Models.html)
3. Scroll down to [Training Algorithm](https://uvadlc-notebooks.readthedocs.io/en/latest/tutorial_notebooks/tutorial8/Deep_Energy_Models.html#Training-algorithm)
4. See error in Algorithm 2 vs code in cell 6.

**Expected behavior**
cdiv_loss = real_out.mean() - fake_out.mean()

**Comment by phlippe** [comment]
Hi, you are right that the algorithm and implementation may first seem to mismatch, but note: the network implements $f_{\theta}(x)$ while the algorithm shows $E_{\theta}(x)$. As explained in the [CNN model](https://uvadlc-notebooks.readthedocs.io/en/latest/tutorial_notebooks/tutorial8/Deep_Energy_Models.html#CNN-Model), we follow the common practice that $f_{\theta}(x)$ models the negative energy, i.e. $-E_{\theta}(x)$. Hence, the code in cell 6 follows the same setup, implying that the loss is indeed fake_out.mean() - real_out.mean().

**Comment by phlippe** [comment]
Fixed with b1636c2a1931a42a80cd445f7ebe7748a6fdc1c6

**Comment by najwalb** [comment]
Thanks for clarifying :)
